app/src/main/java/com/github/se/signify/ui/screens/profile/FriendsListScreen.kt

```diff
@@ -197,6 +212,22 @@ fun RemoveFriendButton(userViewModel: UserViewModel) {
       }
 }
 
+@Composable
+fun MyProfileButton(userViewModel: UserViewModel, navigationActions: NavigationActions) {
+  Button(
+      onClick = {
+        navigationActions.navigateTo(Route.PROFILE)
+        userViewModel.setSearchResult(null)
+      },
+      colors =
+          ButtonDefaults.buttonColors(
+              containerColor = MaterialTheme.colorScheme.primary,
+              contentColor = MaterialTheme.colorScheme.onPrimary),
+      modifier = Modifier.fillMaxWidth()) {
+        Text("My Profile")
+      }
+}
```

Comment:
Hardcoded Text: The text "My Profile" is hardcoded. Consider moving it to a strings.xml file or a constants object for localization and better maintainability.

Null Reset Logic: The line userViewModel.setSearchResult(null) could use a comment or explanation for why the search result is being reset here. This will help other contributors understand its significance.